src/components/navbar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react' 
+import Navbar from 'react-bootstrap/Navbar' 
+import Nav from 'react-bootstrap/Nav' 
+
+function NavbarComp (props){
+
+  const handleSignoutClick = e => {
+    localStorage.removeItem( 'jwt') 
+    localStorage.removeItem( 'currentUser') 
+  }
+
+  const handleStopLink = e => {
+    if (!localStorage.getItem( 'jwt')){
+      document.getElementById( 'home_link').setAttribute( 'href',  '/') 
+    }
+  }
```

Comment:
You could ask for an existing `currentUser` here instead of using the localStorage, and the `useHistory` hook to navigate.